_format/1.1/index.md

```diff
@@ -2326,6 +2356,51 @@ An error object **MAY** have the following members:
 * `meta`: a [meta object][meta] containing non-standard meta-information about the
   error.
 
+## <a href="#appendix" id="appendix" class="headerlink"></a> Appendix
+### <a href="#appendix-query-details" id="appendix-query-details" class="headerlink"></a> Query Parameters Details
+#### <a href="#appendix-query-details-parsing" id="appendix-query-details-parsing" class="headerlink"></a> Parsing/Serialization
+A query parameter is a name–value pair extracted from, or serialized into, a 
+URI's query string. 
+
+To extract the query parameters from a URI, an implementation **MUST** run the 
+URI's query string, excluding the leading question mark, through the 
+[`application/x-www-form-urlencoded` parsing algorithm](https://url.spec.whatwg.org/#urlencoded-parsing),
+with one exception: JSON:API allows the specification that defines a query 
+parameter's usage to provide its own rules for parsing the parameter's value 
+from the `value` bytes identified in steps 3.2 and and 3.3 of the `application/x-www-form-urlencoded` 
+parsing algorithm. The resulting value might not be a string.
```

Comment:
```suggestion
parsing algorithm. The resulting value might not be a string, but could be a (nested) array of strings instead.
```
Small clarification, because "might not be a string" sounds rather ambiguous?

Reply:
The current language is intentionally open ended. 

In https://github.com/json-api/json-api/issues/1266#issuecomment-379600455, I gave the example of a URL like `?sort=distanceFrom(lat,lng),otherField`. If a profile were to specify using the sort parameter like that (which I think would be legal), the parse result for that sort field would likely be some sort of object/tree, so just saying that the result could be a (nested) array of strings is insufficient.

Reply:
👍